functests/utils/nodes/nodes.go

```diff
@@ -19,6 +20,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// NodesSelector represents the label selector used to filter impacted nodes.
+var NodesSelector string
```

Comment:
Any good reason why we use a global? all considered this should work fine, but I dislike globals in general and I'd prefer to actually call getenv() each time MatchingOptionalSelector is invoked.
What do other reviewers think?

Reply:
This is an artifact of the old copy-paste technique. I do prefer to avoid global variables and we can always promote it when/if we will need.

Reply:
@fromanirh thinking a little bit more, keeping it a global may not be such a bad idea. We already have RoleWorkerCNF and PerformanceProfileName globals that do the same. An alternative is to not use RoleWorkerCNF global directly but getting the env variable as part of nodes.GetByRole every time we need.

However the side effect is that one could *change* the env variable during the tests and some the tests *may* run with different values. So using is as a global can be used in order to emphases we only get the env variable once.

Moving it to utils/consts.go

Reply:
Moved and re-pushed